docs/modules/examples/pages/langchain_multimodal_gemini.adoc

```diff
@@ -2,11 +2,11 @@
 
 image::https://colab.research.google.com/assets/colab-badge.svg[align="left",link="https://colab.research.google.com/github/datastax/ragstack-ai/blob/main/examples/notebooks/langchain_multimodal_gemini.ipynb"]
 
-This notebook demonstrates using LangChain, AstraDB and a Google Gemini Pro Vision model to perform multi-modal Retrieval Augmented Generation (RAG).
+This notebook demonstrates using LangChain, Astra DB and a Google Gemini Pro Vision model to perform multi-modal Retrieval Augmented Generation (RAG).
```

Comment:
Should we specify Astra DB Serverless here? Also, use the Oxford comma.

Reply:
Thanks. Done in latest commit.